<?xml version="1.0"?>
<launch>
    <node pkg="visual_servo" type="usb_detect" name="usb_detect" output="screen">
        <remap from="usb_cam/image_raw" to="/image_raw"/> 
        <remap from="usb_cam/camera_info" to="/camera_info"/>
    </node>
</launch>